streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableTransformValues.java

```diff
@@ -141,16 +144,32 @@ public void close() {
 
         @Override
         public void init(final ProcessorContext context) {
+            internalProcessorContext = (InternalProcessorContext) context;
             parentGetter.init(context);
             valueTransformer.init(new ForwardingDisabledProcessorContext(context));
         }
 
         @Override
         public ValueAndTimestamp<V1> get(final K key) {
             final ValueAndTimestamp<V> valueAndTimestamp = parentGetter.get(key);
-            return ValueAndTimestamp.make(
+
```

Comment:
To confirm my understanding, the reason only `KTableTransformValuesGetter` and not any of the other `KTableValueGetter` implementations are impacted by this bug is because only `KTableTransformValuesGetter` gives users access to the `ProcessorContext`. Is that true?

Reply:
Correct.